.cirrus.yml

```
@@ -111,23 +111,23 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_native_tsan.sh"
 
 task:
-  name: '[depends, sanitizers: memory (MSan)] [jammy]'
+  name: '[MSan, depends] [jammy]'
   << : *GLOBAL_TASK_TEMPLATE
   container:
     image: ubuntu:jammy
   env:
     FILE_ENV: "./ci/test/00_setup_env_native_msan.sh"
 
 task:
```

YAMLlint (1.35.1)

[error] duplication of key "task" in mapping

(key-duplicates)